content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -15,63 +15,67 @@ description: Creating Secret objects using resource configuration file.
 
 ## Create the Config file
```

Comment:
I recommend using the correct term in the heading and the text: “manifest” (not “config file”)